libs/base/ui/form/quantity-input/quantity-input.styles.ts

```diff
@@ -20,7 +21,11 @@ export const styles = css`
   oryx-input {
     --oryx-form-control-border-radius: 0;
 
-    width: var(--oryx-cart-quantity-input-width, 71px);
+    ${unsafeCSS(
+      featureVersion >= '1.3'
+        ? 'width: var(--oryx-quantity-input-width, 71px);'
+        : 'width: var(--oryx-cart-quantity-input-width, 71px);'
+    )}
```

Comment:
In this case we avoid some code duplication. But at the same time we decrease readability a bit. So up to you.
```suggestion
      const inputWidth = featureVersion >= '1.3' ? '--oryx-quantity-input-width' : '--oryx-cart-quantity-input-width';
...
    ${unsafeCSS(
      width: var(${inputWidth}, 71px);
    )}
```

Reply:
refactored